docs/guide/http_cache.md

```diff
@@ -53,7 +69,7 @@ fos_http_cache:
                         s_maxage: 20
 ```
 
-Similarly, if you want to apply performance tuning to avoid crawlers affecting the setup too much, you can also set up caching of generic 404s and similar error pages in the following way:
+Similarly, to apply performance tuning to avoid crawlers affecting the setup too much, we also set up caching of generic 404s and similar error pages in the following way:
```

Comment:
```suggestion
Similarly, to apply performance tuning to avoid crawlers affecting the setup too much, set up caching of generic 404s and similar error pages in the following way:
```